Cannot create a codespace

## 🐛 Bug

```
This codespace is currently running in recovery mode due to a container error.
1. Use Cmd/Ctrl + Shift + P -> "Codespaces: View Creation Log" to see full logs
2. Update your devcontainer configuration as needed
3. Use Cmd/Ctrl + Shift + P -> "Codespaces: Rebuild Container" to retry
4. For help, read more about custom configuration: https://aka.ms/ghcs-custom-configuration
mkdir: cannot create directory ‘/root’: Permission denied
codespaces-a25b92:/workspaces/pyodide$ touch: cannot touch '/root/.config/vscode-dev-containers/first-run-notice-already-displayed': Permission denied
```

### To Reproduce

Try to create a codespace

### Expected behavior

A working codespace

### Environment
 
- Commit hash of Pyodide git repository: 10677a475f245ac4fe2d628b1ac46858de22e744

Yes I can confirm this from the error log.

```
[notice] To update, run: pip install --upgrade pip
2025-02-27 12:42:14.663Z: ERROR: file:///workspaces/pyodide/pyodide-build does not appear to be a Python project: neither 'setup.py' nor 'pyproject.toml' found.
2025-02-27 12:42:14.708Z: make: *** [Makefile:229: pyodide_build] Error 1
```

--- a/.devcontainer/devcontainer.json
+++ b/.devcontainer/devcontainer.json
@@ -1,7 +1,8 @@
 {
   "name": "Docker",
   // keep in sync with "run_docker"
-  "image": "pyodide/pyodide-env:20241106-chrome130-firefox132",
+  "image": "pyodide/pyodide-env:20250116-chrome132-firefox134",
   "remoteUser": "root",
+  "initializeCommand": "git submodule update --init",
   "onCreateCommand": ".devcontainer/onCreate-docker.sh"
 }

--- a/.gitpod.yml
+++ b/.gitpod.yml
@@ -10,7 +10,7 @@ github:
 tasks:
   - name: Setup the environment
     init: |
-      PYTHON_VERSION=3.11.3
+      PYTHON_VERSION=3.12.7
       pyenv install $PYTHON_VERSION
       pyenv global $PYTHON_VERSION
       python -m pip install -r requirements.txt

